### DISCUSSION TITLE
can't find the process_outcar_5.3.py file

### DISCUSSION THREAD

**Discussion by phyoung123** [tdep-developers/tdep-tutorials#56] [Q&A]
Hello  @flokno ,

 I used VASP to prepare MD trajectory, but I cannot find the process_outcar.py file to process the data, would you mind showing me where I can find it?

Best regards,
phyoung

**Comment by DeepChoudhuri** [comment]
Same here - I cannot find the script either.

**Comment by flokno** [comment]
We support parsing via ASE:
https://github.com/tdep-developers/tdep-tutorials/tree/main/00_preparation/parsing/VASP
